### PR TITLE
Added set-property command

### PR DESCRIPTION
**Pull request by peetsasaki**

### CODE REVIEW

File: samples/python/pbapi/__init__.py [hunk 1]
```diff
@@ -232,7 +232,10 @@ def list_beacons(self, arguments):
                     .format(next_page_token, beacons_resp['nextPageToken'])
             if next_page_token is not None and beacons_resp['nextPageToken'] == next_page_token:
                 break
-            next_page_token = beacons_resp['nextPageToken']
+            try: 
+                next_page_token = beacons_resp['nextPageToken']
+            except KeyError:
+                break
```
Comment (stefano-maggiolo) [inline samples/python/pbapi/__init__.py]: I suppose you should consume the beacons even if there is no next page token, no?

File: samples/python/pbapi/__init__.py [hunk 2]
```diff
@@ -747,6 +750,56 @@ def bulk_register(self, arguments):
...
+        Accepts a CSV file with a beacon_name column and additional columns whose header is the
+        property name and whose values on each row are the property values.
+
+        Args:
+            arguments: list of arguments passed from CLI. Pass ['--help'] for details.
+
+        Returns:
+            Nothing.
+        """
+        args_parser = argparse.ArgumentParser(description='Sets a property on a beacon.',
```
Comment (stefano-maggiolo) [inline samples/python/pbapi/__init__.py]: "Sets properties on beacons based on the content of a CSV files"?

File: samples/python/pbapi/__init__.py (hunk 1 above)
Comment (peetsasaki) [reply samples/python/pbapi/__init__.py]: Ah, looks like this logic was even more messed up than that. Fixed, I think.

File: samples/python/pbapi/__init__.py (hunk 2 above)
Comment (peetsasaki) [reply samples/python/pbapi/__init__.py]: Done.

**Pull request merged**